mappings/net/minecraft/client/recipe/book/IRecipeDisplayer.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS cwh net/minecraft/client/recipe/book/IRecipeDisplayer
```

Comment:
No I! Also, RecipeDisplayer? Why not RecipeViewer or RecipePainter?

Reply:
It was initially RecipeSeer because all it did was notify the recipe book that a client had already seen the recipe and the book doesn't need to make it bounce up and down.